5calls/app/src/main/java/org/a5calls/android/a5calls/controller/RepCallActivity.java

```diff
@@ -232,7 +232,13 @@ private void setButtonsEnabled(boolean enabled) {
     private void setupContactUi(int index, boolean expandLocalSection) {
         final Contact contact = mIssue.contacts[index];
         contactName.setText(contact.name);
-        contactReason.setText(contact.reason);
+
+        // Set the reason for contacting this rep, using default text if no reason is provided.
+        final String contactReasonText = contact.reason.isEmpty()
```

Comment:
instead of contact.reason.isEmpty(), use TextUtils.isEmpty(contact.reason) just in case it is null.